packages/core/src/codewhisperer/models/model.ts

```diff
@@ -315,6 +316,8 @@ export class TransformByQState {
 
     private dependencyFolderInfo: FolderInfo | undefined = undefined
 
+    private planSteps: TransformationSteps | undefined = undefined
```

Comment:
Not required for this PR: would it make sense to move these gumby-specific state-keeping variables to be in the gumby module, i.e. `amazonqGumby/models/model.ts`? Especially after this change to remove the transform button, they don't seem to be referenced anywhere except by our code

Reply:
Yep that definitely makes sense. My next PR can be just for this one change.